dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -585,6 +585,15 @@
 		voltage-reference = <0>;
 		#io-channel-cells = <1>;
 	};
+
+	vref: vref@111000 {
+		compatible = "nxp,vref";
+		regulator-name = "lpc55s36-vref";
```

Comment:
why use this name? This update is for mcxn94x right?

Reply:
same ip, lpc55s36-vref is applicable for the n10.

Reply:
Sure, but the `compatible` string manages the driver used. The `regulator-name` property is documented as "A string used as a descriptive name for regulator outputs". I think we can change this to "mcxn94x-vref" for this SOC.

Reply:
Yes, which driver to use is determined by the compatible property, the regulator-name property is not for this.